### ISSUE TITLE
Implement "os" collector

### ISSUE THREAD

**Issue by fujimotos** [fluent-plugins-nursery/fluent-plugin-windows-exporter#3]
Prometheus' Windows Exporter

* https://github.com/prometheus-community/windows_exporter/blob/master/collector/os.go
* https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.os.md

**Comment by fujimotos** [comment]
OK. I added first cut at `collect_os` in 087b9330eb1f1919cde68a3b0282e3a63d77f0e0.

Here is how the actual metrics data looks like (tested with Windows Server 2019 on Azure):

```console
$ bundle exec fluend -c test.conf
2021-12-10 01:12:07.111764100 +0000 test.log: {"type":"gauge","name":"windows.os.info","desc":"Windows version info","labels":{"product":"Microsoft Windows Server 2022 Datacenter","version":"10.0.20348"},"value":1.0}
2021-12-10 01:12:07.111764100 +0000 test.log: {"type":"gauge","name":"windows.os.timezone","desc":"OperatingSystem.LocalDateTime","labels":{"timezone":"Coordinated Universal Time"},"value":2195898368}
2021-12-10 01:12:07.111764100 +0000 test.log: {"type":"gauge","name":"windows.os.time","desc":"OperatingSystem.LocalDateTime","labels":{},"value":1639098727.1190197}
2021-12-10 01:12:07.111764100 +0000 test.log: {"type":"gauge","name":"windows.os.paging_free_bytes","desc":"OperatingSystem.FreeSpaceInPagingFiles","labels":{},"value":0}
2021-12-10 01:12:07.111764100 +0000 test.log: {"type":"gauge","name":"windows.os.virtual_memory_bytes","desc":"OperatingSystem.TotalVirtualMemorySize","labels":{},"value":3170181120}
2021-12-10 01:12:07.111764100 +0000 test.log: {"type":"gauge","name":"windows.os.processes_limit","desc":"OperatingSystem.MaxNumberOfProcesses","labels":{},"value":4294967295.0}
2021-12-10 01:12:07.111764100 +0000 test.log: {"type":"gauge","name":"windows.os.processes_limit","desc":"OperatingSystem.TotalVirtualMemorySize","labels":{},"value":140737488224256}
2021-12-10 01:12:07.111764100 +0000 test.log: {"type":"gauge","name":"windows.os.processes","desc":"OperatingSystem.NumberOfProcesses","labels":{},"value":125}
2021-12-10 01:12:07.111764100 +0000 test.log: {"type":"gauge","name":"windows.os.users","desc":"OperatingSystem.NumberOfUsers","labels":{},"value":1}
2021-12-10 01:12:07.111764100 +0000 test.log: {"type":"gauge","name":"windows.os.paging_limit_bytes","desc":"OperatingSystem.SizeStoredInPagingFiles","labels":{},"value":1073479680}
2021-12-10 01:12:07.111764100 +0000 test.log: {"type":"gauge","name":"windows.os.virtual_memory_bytes","desc":"OperatingSystem.TotalVirtualMemorySize","labels":{},"value":5367971840}
2021-12-10 01:12:07.111764100 +0000 test.log: {"type":"gauge","name":"windows.os.visible_memory_bytes","desc":"OperatingSystem.TotalVisibleMemorySize","labels":{},"value":4294496256}
```

The missing bit is `windows.os.paging_free_bytes`, which we need HPD perf data to
compute.

**Comment by fujimotos** [comment]
> The missing bit is windows.os.paging_free_bytes, which we need HPD perf data to compute.

I implemented this in 05455c961732e0bc4511c66dcac2dd41da37ffc0. Here is how the metric looks like:

```ruby
2021-12-13 00:59:27.972306900 +0000 test.log: {"type":"gauge","name":"windows_os_paging_free_bytes",
"desc":"OperatingSystem.FreeSpaceInPagingFiles","labels":{},"value":4096}
```

**Comment by fujimotos** [comment]
Now it reaches the point where the feature set is in parity with `windows-exporter`.
Let's mark this ticket as done.

**Comment by daipom** [comment]
The value of `windows_os_paging_free_bytes` was wrong because of the following problem.

https://github.com/fluent-plugins-nursery/fluent-plugin-windows-exporter/issues/1#issuecomment-994208862

This is fixed in e42b1dd.

**Comment by fujimotos** [comment]
> The value of windows_os_paging_free_bytes was wrong because of the following problem.

@daipom Confirmed. Thank you!
